wolox-books/src/app/store/books.reducer.ts

```diff
@@ -0,0 +1,17 @@
+import { Action } from '@ngrx/store'
+import { Book } from '../models/lists-books.model'
+import * as ListBooksCarActions from './books.actions'
+
+export function reducer(state: Book[] = [], action: ListBooksCarActions.Actions) {
```

Comment:
declare an initial state const in this file, before calling the reducer, you should have a variable inside the state called books, the state should be an object, not an array